service/matching/matchingEngine.go

```diff
@@ -203,6 +203,10 @@ func (e *matchingEngineImpl) getTaskListManager(taskList *taskListID,
 		logger.Info("Task list manager state changed", tag.LifeCycleStartFailed, tag.Error(err))
 		return nil, err
 	}
+	e.metricsClient.Scope(metrics.MatchingTaskListMgrScope).UpdateGauge(
+		metrics.TaskListManagersGauge,
+		float64(len(e.taskLists)),
```

Comment:
this is a racy read, since this is outside the locked scope

Reply:
yep, missed that. thanks!